epub33/fxl-a11y/index.html

```diff
@@ -192,6 +192,8 @@ <h3>Reading order</h3>
 
             <blockquote>A key concept of EPUB is that an EPUB publication consists of multiple resources that can be completely navigated and consumed by a person or program in some specific order. - 1.2.1 Reading Order EPUB 3.3 Overview [[epub-overview-33]]</blockquote>
 
+            <p>Reading order in EPUB refers to the order of content documents in the <a href="https://www.w3.org/TR/epub-33/#sec-pkg-spine"><code>spine</code></a> section of the <a href="https://www.w3.org/TR/epub-33/#sec-package-doc">Package</a> document. However, the concept of reading order is also an important consideration for accessibility and refers to the visual and programmatic order of elements in a content document. In both EPUB and accessibility usage, reading order must be logical and accurate both visually and programmatically.</p>
```

Comment:
Minor nit, but package doesn't need to be capitalized. I'd also link the full term.

```suggestion
            <p>Reading order in EPUB refers to the order of content documents in the <a href="https://www.w3.org/TR/epub-33/#sec-pkg-spine"><code>spine</code></a> section of the <a href="https://www.w3.org/TR/epub-33/#sec-package-doc">package document</a>. However, the concept of reading order is also an important consideration for accessibility and refers to the visual and programmatic order of elements in a content document. In both EPUB and accessibility usage, reading order must be logical and accurate both visually and programmatically.</p>
```